src/RocksDBStore/Plugins/Storage/Store.cs

```diff
@@ -20,8 +20,16 @@ internal class Store : IStore
         public Store(string path)
         {
             var families = new ColumnFamilies();
-            for (int x = 0; x <= byte.MaxValue; x++)
-                families.Add(new ColumnFamilies.Descriptor(x.ToString(), new ColumnFamilyOptions()));
+
+            try
+            {
+                foreach (var family in RocksDb.ListColumnFamilies(Options.Default, Path.GetFullPath(path)))
```

Comment:
How to column families get added after this change? For a newly created RocksDb store, there will be no column families and it doesn't look like you've added a call to CreateColumnFamily anywhere (though I could have missed that)

Reply:
They are created when not found here.
https://github.com/neo-project/neo-modules/blob/496d390f803bfafffc0038867c281ddf80f75664/src/RocksDBStore/Plugins/Storage/Store.cs#L76-L82